### DISCUSSION TITLE
TERKAIT PERFORMA

### DISCUSSION THREAD

**Discussion by rezaawp** [deaafrizal/cuyuniverse-forum#78] [Q&A] [answered]
punten kang, kalo upload foto, foto nya mending di simpan di local storage apa di database ? dan mana performa yang paling kenceng saat web nya sudah punya banyak data ?

**Comment by deaafrizal** [comment, marked as answer]
lebih bagus disimpan ke storage external, contoh: google drive, dropbox, google bucket. di database lebih baik menampung filename nya saja.

**Reply by rezaawp** [reply]
kalo cuyuniverse foto nya disimpen dimana bang? kan kita juga bisa naruh foto di dalem folder project kita, nah klo disitu, worth it ga?

**Reply by deaafrizal** [reply]
kalau masih kecil apps nya aman2 aja ko dan server internalnya cukup gede worth it
